--- a/src/http/ngx_http_header_filter_module.c	2017-09-05 18:09:17.364848277 +0900
+++ b/src/http/ngx_http_header_filter_module.c	2017-09-05 18:10:30.289512878 +0900
@@ -46,9 +46,9 @@
 };
 
 
-static u_char ngx_http_server_string[] = "Server: nginx" CRLF;
-static u_char ngx_http_server_full_string[] = "Server: " NGINX_VER CRLF;
-static u_char ngx_http_server_build_string[] = "Server: " NGINX_VER_BUILD CRLF;
+//static u_char ngx_http_server_string[] = "Server: nginx" CRLF;
+//static u_char ngx_http_server_full_string[] = "Server: " NGINX_VER CRLF;
+//static u_char ngx_http_server_build_string[] = "Server: " NGINX_VER_BUILD CRLF;
 
 
 static ngx_str_t ngx_http_status_lines[] = {
@@ -279,7 +279,7 @@
 
     clcf = ngx_http_get_module_loc_conf(r, ngx_http_core_module);
 
-    if (r->headers_out.server == NULL) {
+    /* if (r->headers_out.server == NULL) {
         if (clcf->server_tokens == NGX_HTTP_SERVER_TOKENS_ON) {
             len += sizeof(ngx_http_server_full_string) - 1;
 
@@ -289,7 +289,7 @@
         } else {
             len += sizeof(ngx_http_server_string) - 1;
         }
-    }
+    } */
 
     if (r->headers_out.date == NULL) {
         len += sizeof("Date: Mon, 28 Sep 1970 06:00:00 GMT" CRLF) - 1;
@@ -448,7 +448,7 @@
     }
     *b->last++ = CR; *b->last++ = LF;
 
-    if (r->headers_out.server == NULL) {
+    /* if (r->headers_out.server == NULL) {
         if (clcf->server_tokens == NGX_HTTP_SERVER_TOKENS_ON) {
             p = ngx_http_server_full_string;
             len = sizeof(ngx_http_server_full_string) - 1;
@@ -463,7 +463,7 @@
         }
 
         b->last = ngx_cpymem(b->last, p, len);
-    }
+    } */
 
     if (r->headers_out.date == NULL) {
         b->last = ngx_cpymem(b->last, "Date: ", sizeof("Date: ") - 1);
